index.html

```diff
@@ -0,0 +1,41 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+    <link rel="stylesheet" href="styles.css">
+</head>
+<body class="page">
+    <header class="header">
+        <img src="images/menu.svg" alt="Menu">
+    </header>
```

Comment:
The `nav` tag for semantics reasons, is more appropriate to enclose the hamburger image since it controls the navigation menu. Kindly enclose the `img` tag within a `nav` tag and both within the `header` tag.